docs/src/components/landing/PricingTable.js

```diff
@@ -286,17 +295,37 @@ function PricingTable({
         <Box sx={gridSx}>
           <Typography />
           {plans.map((plan) => (
-            <Box key={plan} sx={{ display: 'flex', flexDirection: 'column', p: 2 }}>
+            <Box
```

Comment:
Could someone help me out with this particular section? I was going for making only the _recommended_ column highlighted but could figure out a way to do so.

<img width="500" alt="Screen Shot 2023-04-05 at 21 26 43" src="https://user-images.githubusercontent.com/67129314/230242987-b3b0923b-02dc-4363-8346-7d9e21f33b01.png">

Reply:
Added a commit that does this!

Reply:
Thank you, really appreciate it!